libraries/p5videoKit.yaml

```diff
@@ -0,0 +1,11 @@
+name: p5videoKit
+description: A dashboard for mixing video in the browser
+category: video, shaders
```

Comment:
The category will be the header the library appears under on the page, so we're just looking for a single category currently. Is there one of these that you feel is a better fit than the other?

Reply:
The author and I think video would be a better category. Video is not in the category list, could we add it? Thank you!

Reply:
For the initial version of the website we're just going to stick with the current categories so that we make sure we have a few things in each one, but we'll keep it in mind and add it once we get a few video addons! We have one for sound too so it would make sense.

Reply:
thank you for the information! the author would like to put "algorithms" instead since it seems the most generic category. thank you so much!